utilities/generate_api.py

```diff
@@ -40,6 +42,33 @@
 JAVA_GAPIC="java_gapic"
 JAVA_DISCOGAPIC="java_discogapic"
 
+
+def dump_versions(config_path):
+    print("Component versions:")
+
+    print(subprocess.check_output(['artman', '--version'], stderr=subprocess.STDOUT).strip())
+
+    print("googleapis %s" % get_git_repo_version(os.path.dirname(config_path)))
```

Comment:
use the more modern style: `"googleapis {}".format(get...)`

Reply:
done